spring-thrift-starter/src/main/java/info/developerblog/spring/thrift/client/pool/ThriftClientPooledObjectFactory.java

```diff
@@ -74,7 +100,7 @@ public TServiceClient create(ThriftClientKey key) throws Exception {
     public void activateObject(ThriftClientKey key, PooledObject<TServiceClient> p) throws Exception {
         super.activateObject(key, p);
 
-        Optional requestId = Optional.fromNullable(MDC.get(requestIdLogger.getMDCKey()));
+        Optional requestId = Optional.ofNullable(MDC.get(requestIdLogger.getMDCKey()));
```

Comment:
Why not? :) It's for SpringBoot and as I know people who use it don't worry about version of Java. Do you think opposite way?